### ISSUE TITLE
[Feature request] Automatically save .svf after checking files

### ISSUE THREAD

**Issue by typhoon71** [OV2/RapidCRC-Unicode#45]
I'd like to ask for the ability to save the .sfv after checking some files.

For example, I'm checking the crc (crc32) of a bunch of files, and at the same time calculating the .md5: I'd like to have the .md5 saved somewhere, so that the next time I can reuse them to check the files.

Could this be done? Thanks.

**Comment by vatterspun** [comment]
I don't know how OV2 feels about it, but I guess I can see the advantages of hash file generation whatever something is checked.  For example if you wanted to hash a bunch of different folders and didn't want to have to click multiple buttons everytime.

The major issue I can see is that there are so many options for .SFV generation.  For example, when you run the program currently and click "SFV file" you're presented with 4 different options.  Which option would be used?  Where would you add that functionality?

**Comment by typhoon71** [comment]
Well, I would create a new "other crc to calculate" section under "algorithms", by default with no algo selected. That could be the secondary crc to calculate.

Regarding the right click menu, it could be new entries, like the existing ones + secondary crc, or it could be an option to actually calculate those by default. 

Still this is a suggestion, since there's also the time (OV2 time) needed to implement it to consider when thinking how to "integrate" it into Rapidcrc32unicode.

**Comment by vatterspun** [comment]
Right, I just always try to think about where something might go in the interface.  Then, if the dev says "I like the idea but I have no idea where to put it" I can mock something up.
